### PR TITLE
Amended style rules for merchant portal

### PR DESCRIPTION
**Pull request by kelvinma**

### CODE REVIEW

**Comment by phereford** [comment]
What are these rules changing?

File: eslintrc.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "no-alert": 0,
     "no-lone-blocks": 0,
     "jsx-quotes": 1,
+    "max-statements": 30,
```
Comment (kelvinma) [inline eslintrc.json]: max statements/lines per function

File: scss-lint.yml [hunk 2]
```diff
@@ -87,7 +87,7 @@ linters:
   ImportPath:
     enabled: true
     leading_underscore: false
-    filename_extension: false
+    filename_extension: true
```
Comment (kelvinma) [inline scss-lint.yml]: import paths for scss must have extensions

File: scss-lint.yml [hunk 3]
```diff
@@ -215,7 +215,7 @@ linters:
 
   StringQuotes:
     enabled: true
-    style: single_quotes # or double_quotes
+    style: double_quotes # or single_quotes
```
Comment (kelvinma) [inline scss-lint.yml]: all of our stylesheets use double quotes. It would be easier to change the rule than to change every stylesheet in every codebase

**Pull request merged**